packages/common/lib/types/CredentialIssuance.types.ts

```diff
@@ -52,12 +52,34 @@ export type DecodeURIAsJsonOpts = {
 };
 
 export interface JWK {
-  kty?: string;
+  alg?: string;
```

Comment:
The JWK interface does not need to be extended with all options available. We are not using most of them, so this will only be confusing. Since it is an interface anyway the runtime object could still have additional properties, even if we are not using them.

Suggest to create 2 versions. A BaseJWK only containing the values we use and then a sub interface containing everything if we really want to use it in the future

Reply:
`jose` is picking the properties that we use `Pick<JWK, 'kty' | 'crv' | 'x' | 'y' | 'e' | 'n'>`, but I'm fine with anyone. Fixed!